plugins/rollbar/config/initializers/rollbar.rb

```diff
@@ -14,7 +14,6 @@
     config.code_version = Rails.application.config.samson.version&.first(7)
     config.populate_empty_backtraces = true
     config.logger = Rails.logger
-    config.scrub_fields |= Rails.application.config.filter_parameters + ['HTTP_AUTHORIZATION']
```

Comment:
we don't need this since it's rollbar default behavior via https://github.com/rollbar/rollbar-gem/blob/24c4de23c4150d19b408ed14f7839f8cc6e1bc69/lib/rollbar/request_data_extractor.rb#L240